core/src/main/java/org/apache/accumulo/core/clientImpl/InstanceOperationsImpl.java

```diff
@@ -236,12 +246,61 @@ public List<String> getTabletServers() {
   }
 
   @Override
+  public Set<Server> getServers(ServerType type) {
+    Set<Server> results = new HashSet<>();
+    switch (type) {
+      case COMPACTOR:
+        throw new IllegalArgumentException(
+            "Use ExternalCompactionUtil.getCompactorAddrs instead as it will return the compactor grouping");
```

Comment:
ExternalCompactionUtil.getCompactorAddrs is not public API, should not recommend calling it from public API method.

Reply:
Removed this exception in ea67851. Since I added ResourceGroup to Server, I went ahead and implemented this case.